src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -13,6 +13,7 @@
 			<![CDATA[
```

Comment:
Looks like there are some additive changes. Please bump the version to `1.20.1-preview`

Reply:
1.20 is not published yet.

Reply:
No-op in that case